packages/components/atoms/f-form-field/stories/AffixedFormField.stories.js

```diff
@@ -27,8 +30,8 @@ export const AffixedFormFieldComponent = () => ({
         hasError: {
             default: boolean('hasError', false)
         },
-        fieldSize: {
-            default: select('Field Size', VALID_FIELD_SIZES, DEFAULT_FIELD_SIZE)
+        shouldShowLabelText: {
```

Comment:
Can be a separate PR, but should there be a couple of tests for this, or do the current tests cover this new prop?

Reply:
I've added tests now 👍